proto/xudrpc.proto

```diff
@@ -507,8 +507,10 @@ message OpenChannelRequest {
   string node_identifier = 1 [json_name = "node_identifier"];
   // The ticker symbol of the currency to open the channel for.
   string currency = 2 [json_name = "currency"];
-  // The amount of the channel denominated in satoshis.
+  // The amount to be deposited into the channel denominated in satoshis.
   int64 amount = 3 [json_name = "amount"];
+  // The balance amount to be pushed to the remote side of the channel denominated in satoshis.
+  int64 push_amount = 4 [json_name = "push_amount"];
```

Comment:
Why are we using `int64` for the amount again? `uint32` or `uint64` would be better I think since the amount can never be negative

Reply:
You're absolutely right, thanks.